{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"AssertionError: [AssertionError] during step: Update user role on a project via CLI\nMessage:  Expected: ERROR: Illegal argument(s) (Last project admin cannot be demoted), got: Your root certificates are out of date.\nYou are using certifi 2025.7.14, however 2025.8.3 is available.\nPlease consider upgrading certifi package, e.g.:\n    python -m pip install --upgrade certifi\nor\n    conda update certifi\n\nYou are using apolo-extras tool 24.10.0, however 25.8.0 is available.\nERROR: Illegal argument(s) (Last project admin cannot be demoted)","children":[{"name":"Admin demote himself to Manager in project via CLI","uid":"1e5c1e5a41d2c1fc","parentUid":"e99fd585af277dcd69ee7eab786d2445","status":"failed","time":{"start":1756518357244,"stop":1756518387501,"duration":30257},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]}],"uid":"e99fd585af277dcd69ee7eab786d2445"},{"name":"AssertionError: [AssertionError] during step: Remove user from project via CLI\nMessage:  Expected: ERROR: Illegal argument(s) (Last project admin cannot be removed), got: Your root certificates are out of date.\nYou are using certifi 2025.7.14, however 2025.8.3 is available.\nPlease consider upgrading certifi package, e.g.:\n    python -m pip install --upgrade certifi\nor\n    conda update certifi\n\nYou are using apolo-extras tool 24.10.0, however 25.8.0 is available.\nERROR: Illegal argument(s) (Last project admin cannot be removed)","children":[{"name":"Admin removes himself from project via CLI","uid":"739173285547fc76","parentUid":"479dc39cce91958685ded324dcf8bb44","status":"failed","time":{"start":1756519680606,"stop":1756519708813,"duration":28207},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]}],"uid":"479dc39cce91958685ded324dcf8bb44"},{"name":"AssertionError: [AssertionError] during step: Add user not in organization to project via CLI\nMessage: Expected: \nERROR: User 'regression-org-qmkh9szqygwa/regression-21rm208dij' not found in cluster 'default' \nbut got \nYour root certificates are out of date.\nYou are using certifi 2025.7.14, however 2025.8.3 is available.\nPlease consider upgrading certifi package, e.g.:\n    python -m pip install --upgrade certifi\nor\n    conda update certifi\n\nYou are using apolo-extras tool 24.10.0, however 25.8.0 is available.\nERROR: User 'regression-org-qmkh9szqygwa/regression-21rm208dij' not found in cluster 'default'","children":[{"name":"Add user not in organization to project via CLI","uid":"236a153b88a5f73","parentUid":"f0de294adb0bddeb1001d89f3392d5ea","status":"failed","time":{"start":1756521962257,"stop":1756522014250,"duration":51993},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]}],"uid":"f0de294adb0bddeb1001d89f3392d5ea"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}